### PR TITLE
Replace "histogram" with correct duration-type label

### PR DESCRIPTION
**Pull request by ryancastle**
# Changes

- Correct typo in pipelines metrics configuration

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [x] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [x] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by dibyom — APPROVED** [review]

**Comment by afrittoli** [comment]
@ryancastle Thank you for your contribution! As @dibyom said, a precondition for contributing to Tekton is signing the contributor licence agreement. If you are employed, your employer might have a corporate CLA signature in place - if not you can sign it individually using the links in https://github.com/tektoncd/pipeline/pull/5650#issuecomment-1281807924

**Review by bendory — APPROVED** [review]

**Comment by jerop** [comment]
@ryancastle please sign the CLA so we can merge your PR

**Review by jerop — COMMENTED** [review]
Welcome to Tekton @ryancastle!

/meow 
/lgtm

**Comment by vdemeester** [comment]
@ryancastle ping on signing the cla 🙏🏼

File: docs/metrics.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Following values are available in the configmap:
 | metrics.taskrun.duration-type | `histogram` | `tekton_pipelines_controller_pipelinerun_taskrun_duration_seconds` and `tekton_pipelines_controller_taskrun_duration_seconds` is of type histogram |
 | metrics.taskrun.duration-type | `lastvalue` | `tekton_pipelines_controller_pipelinerun_taskrun_duration_seconds` and `tekton_pipelines_controller_taskrun_duration_seconds` is of type gauge |
```
Comment (khrm) [inline docs/metrics.md:51]: Good catch, Can you also update this line also to 
```suggestion
| metrics.taskrun.duration-type | `lastvalue` | `tekton_pipelines_controller_pipelinerun_taskrun_duration_seconds` and `tekton_pipelines_controller_taskrun_duration_seconds` is of type gauge or lastvalue |
```

**Review by khrm — APPROVED** [review]

**Comment by jerop** [comment]
Closing old pull requests, please reopen when you sign the CLA

**Pull request closed without merging**